### DISCUSSION TITLE
Ctrl K+D 格式化命令 在.axaml中不起作用

### DISCUSSION THREAD

**Discussion by AlvinRey** [BYJRK/DotNet-Discussions#197] [其他框架]
如题，Ctrl K+D 格式化命令 在.axaml中不起作用，但在后置代码中(.axaml.cs)能正常使用，为什么呢？是要安装其他扩展去支持该功能吗？

**Comment by BYJRK** [comment]
你是不是没安装插件？因为 axaml 这个文件格式默认 VS 是不认的。需要装了插件之后，VS 才会把它当作 XML（是的，不是 XAML）。不过 XAML Styler 是支持这个格式的。

**Reply by AlvinRey** [reply]
是要安装XAML Styler这个插件吗？还是别的（叫啥）？

**Reply by AlvinRey** [reply]
我装了XAML Styler之后，Ctrl K+D 好像还是没有什么作用。
这是我XAML Styler的设置
![image](https://github.com/BYJRK/DotNet-Discussions/assets/56423507/f40959e7-b1a3-4ccc-8d01-bf6829df8408)

**Reply by AlvinRey** [reply]
好像可以了。。
